main.js

```diff
@@ -33,9 +33,14 @@ window.addEventListener('load', (event) => {
         }
     }
 
-    // TODO: shuffle the deck to be in a random order, see:
-    // https://medium.com/@nitinpatel_20236/how-to-shuffle-correctly-shuffle-an-array-in-javascript-15ea3f84bfb
-
+    // shuffle the deck to be in a random order
+    
```

Comment:
the convention when comments refer to a code block is to have no space (no empty lines) between the comment and the code block it refers to below

Reply:
Okay I sent you a new request to review